third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -145,6 +145,20 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
             dump_bytecode: DumpLevel::None,
             dump_bytecode_filter: None,
         },
+        TestConfig {
+            name: "unused-assignment",
+            runner: |p| run_test(p, get_config_by_name("unused-assignment")),
+            include: vec!["/unused-assignment/"],
+            exclude: vec![],
+            exp_suffix: None,
+            options: opts
+                .clone()
+                .set_experiment(Experiment::UNUSED_ASSIGNMENT_CHECK, true),
+            stop_after: StopAfter::BytecodePipeline(Some("UnusedAssignmentChecker")),
+            dump_ast: DumpLevel::None,
+            dump_bytecode: DumpLevel::EndStage,
```

Comment:
Do we need the bytecode to be dumped for these tests? It seems like a distraction from reviewing the warnings.

Reply:
Done. Was using it for debugging.